src/components/player/Player.jsx

```diff
@@ -0,0 +1,17 @@
+
+export default function Player({
+    player: { name, position, teams }, showDetail = false,
```

Comment:
what this showDetail prop is doing is so cool!

Reply:
stole it from the library demo cuz i also thought it was cool hehe